auth/auth.css

```diff
@@ -0,0 +1,45 @@
+.auth-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 4rem;
+}
+
+.auth-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  width: 25rem;
+  padding: 2rem;
+  box-shadow: 0 0 4px 0px var(--dark-gray-color);
+}
+
+.auth-title {
+}
```

Comment:
can remove this since there are no properties applied

Reply:
Thanks Srikar, done.